README.md

```diff
@@ -194,9 +194,27 @@ for iter = 1:10
 end
 ```
 
+### Parallel processes
+
+For processes that are parallelized with `Threads.@spawn` or `Threads.@threads`,
```

Comment:
What would an example with `Threads.@spawn` look like?

Reply:
just testing.. getting some odd behavior on a basic loop on 1.3-rc4.. `@spawn` is only using 1 thread, even though `nthreads()==5`
```
@show Threads.nthreads() # =5
t = Threads.@spawn for i = 1:100
    println("$i on $(Threads.threadid())")
    sleep(0.1)
end
```

Reply:
Just realized I'm misusing `@spawn`. I'm working on an example

Reply:
@KristofferC that's proving to be a good question.. I can't figure out an example, so I'll remove mention of `@spawn`